docs/source/guides/quickstart.md

```diff
@@ -1,48 +1,195 @@
 # NuCypher Quickstart
 
-## Ursula
 
-### Install NuCypher
+## A Note about Side Channels
+
+The NuCypher network does not store or handle an application's data; instead - it manages access *to* application data.
+Management of encrypted secrets and public keys tends to be highly domain-specific - The surrounding architecture
+will vary greatly depending on the throughput, sensitivity, and sharing cadence of application secrets.
+In all cases, NuCypher must be integrated with a storage and transport layer in order to function properly.
+Along with the transport of ciphertexts, a nucypher application also needs to include channels for Alice and Bob 
+to discover each other's public keys, and provide policy encrypting information to Bob and Enrico.
+ 
+##### Side Channel Data:
+  - Secrets:
+    - Message Kits (Ciphertext)
+  - Identities:
+    - Alice Verifying Key
+    - Bob Encrypting Key
+    - Bob Verifying Key
+  - Policies:
+    - Policy Encrypting Key 
+    - Labels
+
+## Running an Ethereum Node
+
+Operation of a decentralized NuCypher character [`Alice`, `Bob`, `Ursula`] requires
+a connection to an Ethereum node and wallet.
+
+To run a Goerli-connected Geth node in *fast* syncing mode:
 
 ```bash
-$ pip3 install -U nucypher
+$ geth --goerl
 ```
 
-### Run a Federated-Only Development Ursula
+To run a Goerli-connected Geth node in *light* syncing mode:
 
 ```bash
-$ nucypher ursula run --dev --federated-only
+$ geth --goerl --syncmode light
 ```
 
-### Configure a Persistent Ursula
+Note that using `--syncmode light` is not 100% stable but can be a life savior when using 
+a mobile connection (or congested hackathon wifi...).
 
+### Provider URI
+
+Nucypher uses the node's IPC-File to communicate, specified by `provider_uri`.
+By default in ubuntu, the path is `~/.ethereum/goerli/geth.ipc` - This path
+will also be logged to the geth-running console on startup. 
+
+Connect to the Geth Console to test your ethereum node's IPC:
 ```bash
-$ nucypher ursula init --federated-only
+$ geth attach ~/.ethereum/goerli/geth.ipc
 ```
 
-### Run a Persistent Ursula
+### Wallets
+
+To list available accounts on your geth node:
 
 ```bash
-$ nucypher ursula run --network <NETWORK_DOMAIN> --teacher <SEEDNODE_URI> --federated-only
+$ geth attach ~/.ethereum/goerli/geth.ipc
+> eth.accounts
+["0x287a817426dd1ae78ea23e9918e2273b6733a43d", "0xc080708026a3a280894365efd51bb64521c45147"]
 ```
```

Comment:
We probably can also mention that we can create accounts here with `personal.newAccount()`

Reply:
:heavy_check_mark: